animation_nodes/nodes/matrix/distribute_matrices.pyx

```diff
@@ -18,6 +18,7 @@ modeItems = [
     ("MESH", "Mesh", "", "NONE", 3),
     ("SPIRAL", "Spiral", "", "NONE", 4),
     ("SPLINE", "Spline", "", "NONE", 5),
+    ("HEXAGON", "Hexagon", "", "NONE", 6),
```

Comment:
Better call this "Gexagonal Grid" for clarity.

Reply:
Renamed